tests/utils/getScenarioContent.test.tsx

```diff
@@ -1,98 +1,76 @@
+    expect(scenarioTypeWithout).toBe(ScenarioType.Scenario1)
   })
 
-  it('is returned when there are pending weeks and pendingDetermination is null', () => {
-    const basePendingScenarioNull = { hasPendingWeeks: true, pendingDetermination: null }
-    const scenarioType: ScenarioType = getScenario(basePendingScenarioNull)
-    expect(scenarioType).toBe(ScenarioType.BasePending)
+  it('is not returned if pendingDetermination is null', () => {
+    const pendingDeterminationScenarioNull = { pendingDetermination: null }
+    const scenarioTypeNull = getScenario(pendingDeterminationScenarioNull)
+    expect(scenarioTypeNull).not.toBe(ScenarioType.Scenario1)
   })
 
-  it('is returned when there are pending weeks and pendingDetermination is an empty array', () => {
-    const basePendingScenarioEmpty = { hasPendingWeeks: true, pendingDetermination: [] }
-    const scenarioType: ScenarioType = getScenario(basePendingScenarioEmpty)
-    expect(scenarioType).toBe(ScenarioType.BasePending)
+  it('is not returned if pendingDetermination is an empty array', () => {
+    const pendingDeterminationScenarioEmpty = { pendingDetermination: [] }
+    const scenarioTypeEmpty = getScenario(pendingDeterminationScenarioEmpty)
+    expect(scenarioTypeEmpty).not.toBe(ScenarioType.Scenario1)
   })
 })
 
-// Test getScenario(): base state with no pending weeks scenario
-describe('The base state (with no pending weeks) scenario', () => {
-  it('is returned when there are no pending weeks', () => {
-    const scenarioType: ScenarioType = getScenario(baseNoPendingScenario)
-    expect(scenarioType).toBe(ScenarioType.BaseNoPending)
-  })
-
-  it('is returned when there are no pending weeks and pendingDetermination is null', () => {
-    const baseNoPendingScenarioNull = { hasPendingWeeks: false, pendingDetermination: null }
-    const scenarioType: ScenarioType = getScenario(baseNoPendingScenarioNull)
-    expect(scenarioType).toBe(ScenarioType.BaseNoPending)
-  })
-
-  it('is returned when there are no pending weeks and pendingDetermination is an empty array', () => {
-    const baseNoPendingScenarioEmpty = { hasPendingWeeks: false, pendingDetermination: [] }
-    const scenarioType: ScenarioType = getScenario(baseNoPendingScenarioEmpty)
-    expect(scenarioType).toBe(ScenarioType.BaseNoPending)
-  })
-})
-
-// Test getScenario(): error
-describe('Getting the scenario', () => {
-  it.skip('errors when given an unknown scenario', () => {
-    expect(() => {
-      getScenario({})
-    }).toThrowError('Unknown Scenario')
+// Scenarios 7-10
+describe('The Base State scenarios', () => {
+  it('are returned as expected', () => {
+    const baseScenarios = [
+      ScenarioType.Scenario7,
+      ScenarioType.Scenario8,
+      ScenarioType.Scenario9,
+      ScenarioType.Scenario10,
+    ]
+    for (const scenarioType of baseScenarios) {
+      expect(getScenario(apiGatewayStub(scenarioType))).toBe(scenarioType)
+    }
```

Comment:
Note: This is a basically just testing that `getScenario()` can process the output for our stub, so I added tests for our stub.